lib/trento.pm

```diff
@@ -826,6 +828,43 @@ sub cluster_wait_status {
     die "Timeout waiting for the change" if !$done;
 }
 
+=head3 cluster_wait_status_by_regex
+
+Remotly run 'SAPHanaSR-showAttr' in a loop on $host, wait output that match using regular expression
+=cut
+
+sub cluster_wait_status_by_regex {
+    my ($host, $timeout, $regular_expression, $text_to_find) = @_;
```

Comment:
This is related to the comment from @mpagot in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16993/files#r1197935655

Some of the parameters are clear (`$host`, `$timeout`, `$text_to_find`), but the third one isn't, so either rename it so its purpose it's clear, or add a summary of what each is used for as suggested by Michele.

Additionally, are any of the parameters mandatory or can they be dropped? The line below seems to imply that `$timeout` is not always mandatory as it has a default value, but then if `$host`, `$regular_expression` and `$text_to_find` are not mandatory, there is no check in the code verifying that they actually contain something.

I suppose that if either `$regular_expression` or `$text_to_find` are empty or undef, then the function will not do anything.